client/src/redux/modals/modals.types.ts

```diff
@@ -0,0 +1,17 @@
+import { PopperProps } from '@material-ui/core';
```

Comment:
PopperProps works good for the dropdowns since it provides all the necessary types and props required.
It also works for the dialog components since the anchorEl, child, and placement props can be ignored. But the open prop is still required